source/agora/test/Flash.d

```diff
@@ -576,6 +576,13 @@ unittest
     writefln("Sleeping for 7 seconds..");
     Thread.sleep(7.seconds);
 
+    // Second payment attempt should fail because of insufficient funds
+    auto inv_2 = charlie.createNewInvoice(Amount(4_000), time_t.max, "payment 2");
+    alice.payInvoice(inv_2);
+
+    writefln("Sleeping for 7 seconds..");
+    Thread.sleep(7.seconds);
```

Comment:
payInvoice is now blocking so I think we can get rid of the sleep calls between the payInvoice calls in this second test. I did get rid of them in the first test but I forgot about the second one.

Reply:
Done